src/transmitter/types.ts

```diff
@@ -33,7 +33,7 @@ export interface IKubernetesMonitorMetadata {
   namespace?: string;
 }
 
-export interface IDepGraphPayload {
+export interface IDependencyGraphPayload {
```

Comment:
this is fine - just note that DepGraph is also used elsewhere
https://github.com/snyk/dep-graph